src/test/resources/test-export-realm.json

```diff
@@ -1279,354 +1272,553 @@
         }
       ]
     },
     {
-      "id":"c1fca724-cd07-426d-b402-7e0966195097",
-      "alias":"clients",
-      "description":"Base authentication for clients",
-      "providerId":"client-flow",
-      "topLevel":true,
-      "builtIn":true,
-      "authenticationExecutions":[
-        {
-          "authenticator":"client-secret",
-          "requirement":"ALTERNATIVE",
-          "priority":10,
-          "userSetupAllowed":false,
-          "autheticatorFlow":false
-        },
-        {
-          "authenticator":"client-jwt",
-          "requirement":"ALTERNATIVE",
-          "priority":20,
-          "userSetupAllowed":false,
-          "autheticatorFlow":false
+      "id": "cf2929fe-03a4-4cce-a708-11469c40099c",
+      "alias": "User creation or linking",
+      "description": "Flow for the existing/non-existing user alternatives",
+      "providerId": "basic-flow",
+      "topLevel": false,
+      "builtIn": true,
+      "authenticationExecutions": [
+        {
+          "authenticatorConfig": "create unique user config",
+          "authenticator": "idp-create-user-if-unique",
+          "requirement": "ALTERNATIVE",
+          "priority": 10,
+          "userSetupAllowed": false,
+          "autheticatorFlow": false
+        },
+        {
+          "requirement": "ALTERNATIVE",
+          "priority": 20,
+          "flowAlias": "Handle Existing Account",
+          "userSetupAllowed": false,
+          "autheticatorFlow": true
         }
       ]
     },
     {
-      "id":"bf1d7d12-0b33-491d-96d4-fb380b1aa4d6",
-      "alias":"direct grant",
-      "description":"OpenID Connect Resource Owner Grant",
-      "providerId":"basic-flow",
-      "topLevel":true,
-      "builtIn":true,
-      "authenticationExecutions":[
-        {
-          "authenticator":"direct-grant-validate-username",
-          "requirement":"REQUIRED",
-          "priority":10,
-          "userSetupAllowed":false,
-          "autheticatorFlow":false
-        },
-        {
-          "authenticator":"direct-grant-validate-password",
-          "requirement":"REQUIRED",
-          "priority":20,
-          "userSetupAllowed":false,
-          "autheticatorFlow":false
-        },
-        {
-          "authenticator":"direct-grant-validate-otp",
-          "requirement":"OPTIONAL",
-          "priority":30,
-          "userSetupAllowed":false,
-          "autheticatorFlow":false
+      "id": "47bfaadf-233d-42df-907e-f02585f72128",
+      "alias": "Verify Existing Account by Re-authentication",
+      "description": "Reauthentication of existing account",
+      "providerId": "basic-flow",
+      "topLevel": false,
+      "builtIn": true,
+      "authenticationExecutions": [
+        {
+          "authenticator": "idp-username-password-form",
+          "requirement": "REQUIRED",
+          "priority": 10,
+          "userSetupAllowed": false,
+          "autheticatorFlow": false
+        },
+        {
+          "requirement": "OPTIONAL",
```

Comment:
This, and other "requirement": "OPTIONAL" statements should be renamed to "requirement": "CONDITIONAL",